src/App/components/Button/index.js

```diff
@@ -44,7 +44,7 @@ const Button = ({ label, id, name, className, value, href, icon, loading, type,
 
     return (
         <button className={btnClasses} {...attrs}>{icon ? "\n\t\t" : null}
-            {icon && !iconAfter ? <><i className="material-icons" aria-hidden="true">{icon}</i>{"\n\t"}</> : null}
+            {icon && !iconAfter ? <><i className="material-icons-outlined" aria-hidden="true">{icon}</i>{"\n\t"}</> : null}
```

Comment:
Pixel-jocker incoming!
Margin on left and right side of icon are not equal
![image](https://user-images.githubusercontent.com/44967773/184600986-d5e51a97-36d5-4edd-a010-2b4d9f403a3a.png)